webrtc.html

```diff
@@ -3472,6 +3485,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               framerate, <code>degradationPreference</code> indicates what should be prefered.
             </p>
           </dd>
+
+          <dt>DOMString rid</dt>
+          <dd>
+            <p> If set, this RTP encoding will be sent with the RID header extension as
+              defined by [[!RTCWEB-JSEP]].
```

Comment:
Here I'm concerned about applications choosing for collisions, or choosing identifiers that are too long for good performance.